pinot-core/src/main/java/com/linkedin/pinot/core/realtime/impl/kafka/KafkaBrokerWrapper.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (C) 2014-2016 LinkedIn Corp. (pinot-core@linkedin.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.linkedin.pinot.core.realtime.impl.kafka;
+
+import java.lang.reflect.Method;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import kafka.cluster.Broker;
+
+/**
+ * This is a walk around for kafka 8 and 9.
```

Comment:
typo workaround. Can you add some comments around incompatibility issues between 0.8 server and 0.9 consumer.